src/python/pants/backend/terraform/target_types.py

```diff
@@ -18,5 +18,18 @@ class TerraformModule(Target):
     help = """A single Terraform module."""
```

Comment:
If you do end up renaming Sources for `terraform_module`, it'd also be helpful to change this help:

```suggestion
    help = (
        "A single Terraform module corresponding to a directory.\n\n"
        "There must only be one `terraform_module` in a directory.\n\n"
        "Use `terraform_modules` to generate `terraform_module` targets for less boilerplate."
    )
```

Reply:
Will take the change.